django/thunderstore/community/api/experimental/tests/test_api_package_listing.py

```diff
@@ -34,18 +34,27 @@ def test_api_experimental_package_listing_update_user_types(
         TestUserTypes.site_admin: True,
         TestUserTypes.superuser: True,
     }
+
+    error_message = {
+        TestUserTypes.no_user: ["Must be authenticated"],
+        TestUserTypes.unauthenticated: ["Must be authenticated"],
+        TestUserTypes.regular_user: ["User is missing necessary roles or permissions"],
+        TestUserTypes.deactivated_user: ["User has been deactivated"],
+        TestUserTypes.service_account: [
+            "Service accounts are unable to perform this action"
+        ],
+        TestUserTypes.site_admin: None,
+        TestUserTypes.superuser: None,
+    }
```

Comment:
So specifically these errors are probably something we shouldn't by default just "leak" out as API errors by default.